openbb_terminal/cryptocurrency/onchain/onchain_controller.py

```diff
@@ -427,6 +430,50 @@ def call_whales(self, other_args: List[str]):
                 else None,
             )
 
+    @log_start_end(log=logger)
+    def call_topledger(self, other_args: List[str]):
```

Comment:
we can discuss more about this but i think would be cool to have the the command called something like tlquery or something like that because we have a special place to add the Source (which would clearly say TopLedger)